.github/workflows/deploy_docs_and_viewer.yml

```diff
@@ -0,0 +1,26 @@
+name: deploy docs and viewer
+on:
+    workflow_dispatch:
+        inputs:
+          args:
+            description: 'Arguments to the remote script'
+            required: false
+            type: string
```

Comment:
I don't think we will need any remote scripts here since we will only be deploying the built viewer.